core/src/main/java/nl/nn/adapterframework/doc/IbisDocPipe.java

```diff
@@ -15,328 +15,30 @@
-import nl.nn.adapterframework.doc.objects.DigesterXmlHandler;
-import nl.nn.adapterframework.doc.objects.IbisBean;
-import nl.nn.adapterframework.doc.objects.IbisMethod;
-import nl.nn.adapterframework.doc.objects.SchemaInfo;
-import nl.nn.adapterframework.doc.objects.SpringBean;
+import nl.nn.adapterframework.doc.objects.DocInfo;
 import nl.nn.adapterframework.parameters.ParameterList;
 import nl.nn.adapterframework.parameters.ParameterValueList;
 import nl.nn.adapterframework.pipes.FixedForwardPipe;
 import nl.nn.adapterframework.stream.Message;
-import nl.nn.adapterframework.util.ClassUtils;
-import nl.nn.adapterframework.util.LogUtil;
-import nl.nn.adapterframework.util.Misc;
-import nl.nn.adapterframework.util.XmlBuilder;
-import nl.nn.adapterframework.util.XmlUtils;
 
 /**
  * Generate documentation and XSD for code completion of beautiful Ibis configurations in Eclipse
  *
  * @author Jaco de Groot
  */
 public class IbisDocPipe extends FixedForwardPipe {
-	private static Logger log = LogUtil.getLogger(IbisDocPipe.class);
-	private static Set<String> excludeFilters = new TreeSet<String>();
-	static {
-		// Exclude classes that will give conflicts with existing, non-compatible bean definition of same name and class
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.esb\\.WsdlGeneratorPipe");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.ifsa\\.IfsaRequesterSender");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.ifsa\\.IfsaProviderListener");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.sap\\.jco2\\.SapSender");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.sap\\.jco2\\.SapListener");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.sap\\.jco2\\.SapLUWManager");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.sap\\.jco3\\.SapSender");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.sap\\.jco3\\.SapListener");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.sap\\.jco3\\.SapLUWManager");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.pipes\\.CommandSender");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.pipes\\.EchoSender");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.pipes\\.FixedResultSender");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.pipes\\.LogSender");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.pipes\\.MailSender");
-		excludeFilters.add(".*\\.IbisstoreSummaryQuerySender");
-		// Exclude classes that cannot be used directly in configurations
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.pipes\\.MessageSendingPipe");
-		
-		// Exclude classes that should only be used in internal configurations
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.doc\\.IbisDocPipe");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.webcontrol\\..*");
-		excludeFilters.add("nl\\.nn\\.adapterframework\\.pipes\\.CreateRestViewPipe");
```

Comment:
Ik zie dat excludeFilters naar DocInfo verplaatst is, maar ik vind deze configuratie van excludes niet meer terug. Waar is die gebleven? Of is die niet meer nodig?

Reply:
Class nl.nn.adapterframework.doc.InfoBuilderSource